invenio_communities/templates/semantic-ui/invenio_communities/header.html

```diff
@@ -0,0 +1,34 @@
+{# -*- coding: utf-8 -*-
+
+    This file is part of Invenio.
+    Copyright (C) 2022 CERN.
+  
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+  #}
+  
+  {% extends config.THEME_HEADER_TEMPLATE %}
+  
+
+  {%- block navbar_search %}
+    {% set defaultOption = 
+        {
+        "key": "records",
+        "text": _("All") + " " + config.THEME_SITENAME,
+        "value": config.APP_RDM_ROUTES.record_search,
+        }
+    %}
+    {% set options = [defaultOption] %}
+    {% if community %}
+        {% set community_url = config.RDM_COMMUNITIES_ROUTES["community-detail"].replace("<pid_value>", community.slug) %}
+        {% set communityOption = 
+        {
+            "key": "communities",
+            "text": _("In this community"),
+            "value": community_url,
+        }
+        %}
+        {% set options = [communityOption, defaultOption] %}
+    {% endif %}
+    {{ nav_searchbar(options) }}
```

Comment:
this macro is coming from invenio-app-rdm and unfortunately it cannot be used in here. If for some reason we install invenio-communities to any other app (ils?) this will fail since the macro will not be defined

Reply:
Right! 
Would it make sense to move the macro to `invenio-theme`? or any better proposals?

Reply:
in this case we would have to move the entire searchbar to invenio-theme, otherwise the macro would not have a reason to exist there. I simplified the implementation for now, it can be rechallenged though